src/03-ticket-calculator.js

```diff
@@ -109,7 +138,59 @@ function calculateTicketPrice(ticketData, ticketInfo) {}
     purchaseTickets(tickets, purchases);
     //> "Ticket type 'discount' cannot be found."
  */
-function purchaseTickets(ticketData, purchases) {}
+function purchaseTickets(ticketData, purchases) {
+  let welcome = `Thank you for visiting the Dinosaur Museum!\n-------------------------------------------\n`;
+  let totalCost = 0;
+  
+  
+  function printOurPurchaseInfo(price, purchase) {
```

Comment:
put this outside the function it is used. in - this is almost just as hard to read